scripts/run-executable.ps1

```diff
@@ -382,6 +447,23 @@ function Wait-Executable($Exe) {
             LogErr "Process had nonzero exit code: $($Exe.Process.ExitCode)"
             $KeepOutput = $true
         }
+
+        # When using ProcDump, give it a moment to finish writing dumps
+        if ($UseProcDump -and $IsWindows) {
+            Write-Host "Waiting for dump files to be written..."
+            Start-Sleep -Seconds 2
```

Comment:
Is 2 seconds enough? Can't we wait more, and only if the exit code was non-zero?